From 04fc8ace1807a07fb9c6ea557ac60f0cc2e5f6eb Mon Sep 17 00:00:00 2001
From: Ikey Doherty <ikey@solus-project.com>
Date: Sat, 15 Oct 2016 22:19:10 +0100
Subject: [PATCH 3/4] schemas: Add override for the clock settings

 - Hide the date by default
 - Hide weather icon
 - Hide temperature

Signed-off-by: Ikey Doherty <ikey@solus-project.com>
---
 schemas/mate/Makefile.am                                  | 1 +
 schemas/mate/org.mate.panel.applet.clock.gschema.override | 4 ++++
 2 files changed, 5 insertions(+)
 create mode 100644 schemas/mate/org.mate.panel.applet.clock.gschema.override

diff --git a/schemas/mate/Makefile.am b/schemas/mate/Makefile.am
index 6f9184d..a411973 100644
--- a/schemas/mate/Makefile.am
+++ b/schemas/mate/Makefile.am
@@ -6,6 +6,7 @@ MATE_SCHEMAS = \
 	org.mate.peripherals-touchpad.gschema.override \
 	org.mate.Marco.general.gschema.override \
 	org.mate.panel.gschema.override \
+	org.mate.panel.applet.clock.gschema.override \
 	org.mate.panel.menubar.gschema.override \
 	org.mate.NotificationDaemon.gschema.override
 
diff --git a/schemas/mate/org.mate.panel.applet.clock.gschema.override b/schemas/mate/org.mate.panel.applet.clock.gschema.override
new file mode 100644
index 0000000..e627397
--- /dev/null
+++ b/schemas/mate/org.mate.panel.applet.clock.gschema.override
@@ -0,0 +1,4 @@
+[org.mate.panel.applet.clock]
+show-date=false
+show-weather=false
+show-temperature=false
-- 
2.10.1

